### DISCUSSION TITLE
How I can disable SSL verification completely

### DISCUSSION THREAD

**Discussion by angelside** [pdm-project/pdm#1738] [Q&A]
They changed the firewall at work, now almost every package manager has a SSL error. _(I fixed git, go, deno)_ I disabled most of them. How I can disable for PDM?

```
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get
local issuer certificate (_ssl.c:997)
```

Below does not effect

```
[pypi]
verify_ssl = false
```

**Comment by akaIDIOT** [comment]
Found this post trying to find the exact same thing (corporate firewalls be damned 😠), but the configuration setting you posted does work for me. `pdm config` shows it overridden from the default config:

```console
$ pdm config
Site/default configuration
build_isolation = True
[...]

Home configuration ([...]/.config/pdm/config.toml):
check_update = False
pypi.verify_ssl = False
```

After which the exact same error you posted goes away for me (using `pdm lock` for me, but I don't think that would make a difference). 

Hope that helps!
